packages/@aws-cdk/aws-apigatewayv2-integrations/lib/http-private.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  HttpConnectionType,
+  HttpIntegrationType,
+  HttpRouteIntegrationBindOptions,
+  HttpRouteIntegrationConfig,
+  IHttpRouteIntegration,
+  PayloadFormatVersion,
+  HttpMethod,
+  IVpcLink,
+} from '@aws-cdk/aws-apigatewayv2';
+import * as ec2 from '@aws-cdk/aws-ec2';
+
+
+/**
+ * Properties to initialize a new `HttpProxyPrivateIntegration`.
+ */
+export interface HttpProxyPrivateIntegrationProps {
```

Comment:
Same about naming here: can this just be `HttpPrivateIntegrationProps` and `HttpPrivateIntegration`?